src/components/ArticleCard.jsx

```diff
@@ -1,7 +1,7 @@
-const ArticleCard = ({title, author, topic, votes, comment_count, date}) => {
+const ArticleCard = ({article_id, title, author, topic, votes, comment_count, date}) => {
     const formatDate = date.slice(0, -14)
     return (
-        <section className="article-card">
+        <section className="article-card" key={article_id}>
```

Comment:
you shouldn't need a `key` prop here as you've already added one in `Articles.jsx` 🤔